src/Validator/ViolationCollection.php

```diff
@@ -27,6 +27,15 @@ public function getMessages(): StringCollection
 
         return new StringCollection($messages);
     }
+    public function getFormattedMessages(): StringCollection
+    {
+        /** @var array<string> $messages */
+        $messages = $this->map(static function (ViolationInterface $violation) {
```

Comment:
PHPStan dont expect a return type for this closure?

Reply:
Somehow it does not, but I don't know why. I'll add it.